web/src/client/storage.js

```diff
@@ -238,6 +245,257 @@ class ProposalManager {
   }
 }
 
+/**
+ * Class providing an API for managing Direct Access Storage Devices (DASDs)
+ */
+class DASDManager {
+  /**
+   * @param {string} service - D-Bus service name
+   * @param {string} address - D-Bus address
+   */
+  constructor(service, address) {
+    this.service = service;
+    this.address = address;
+    this.proxies = {};
+  }
+
+  /**
+   * @return {DBusClient} client
+   */
+  client() {
+    // return this.assigned_client;
+    if (!this._client) {
+      this._client = new DBusClient(this.service, this.address);
+    }
+
+    return this._client;
+  }
+
+  async isSupported() {
+    const proxy = await this.managerProxy();
```

Comment:
For next iterations: we should use info from ObjectManager instead of trying to create the proxy.